distribution/docs/src/main/resources/_contents/extending-contents.adoc

```diff
@@ -1679,7 +1679,7 @@ This XML file is found inside of the STS bundle and is named `ws-trust-1.4-servi
     </wsp:Policy>
     <wsdl:service name="SecurityTokenService">
         <wsdl:port name="STS_Port" binding="tns:STS_Binding">
-            <soap:address location="http://localhost:8181/services/SecurityTokenService"/>
+            <soap:address location="${public_url}/services/SecurityTokenService"/>
```

Comment:
This should be the secure url. Might want to just grab the latest wsdl for the STS, I'm sure this is out of date anyways.